### PR TITLE
Accessibility configuration

### PR DESCRIPTION
**Pull request by AnsarIbrahim**
### This PR adds the following sections :

- Page titles - no issues found
- Image text -  issues fixed
- Text headings - no issues found
- Color contrast - no issues found
- Resize - no issues found
- Interaction - issues fixed
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,11 +51,11 @@ <h1 class="headline-1-h1">I’m Mariane
...
-                <li><a href="#"><img alt="" src="icons/Vector1.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Linkedin icon.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Vector3.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Vector2.svg"></a></li>
-                <li><a href="#"><img alt="" src="icons/Vector4.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector1.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Linkedin icon.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector3.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector2.svg"></a></li>
+                <li><a href="#"><img alt="social media icon" src="icons/Vector4.svg"></a></li>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.

image
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (yin-ka) [reply index.html]: @ShadyShawkat 

> * [ ]  Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project.
> 
> image image

@ShadyShawkat can you please explain how to fix this issue as we still don't understand the issue.

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @AnsarIbrahim, @yin-ka 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ correct use of gitfhub flow
✔️ linters are passing
✔️ professional documentation
## Optional suggestions

- [ ] When you pass the live link of your portfolio to the site added by the previous reviewer, there is a contrast issue in your buttons. You can use [this site](https://dequeuniversity.com/color-contrast) and add your button hex colors to compare them and get the best results playing with colors. This is for people with low vision who require sufficient contrast. However, since you copied the colors from the figma design. I am approving it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tugsem) [reply index.html]: @yin-ka Kindly read my suggestion for the answer.

**Pull request merged**